### PR TITLE
update explorer

### PR DESCRIPTION
**Pull request by foesa-yang**

### CODE REVIEW

File: docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 # Deploy Explorer
 
-This topic describes how to deploy Explorer locally by RPM, and tar package.
+This topic describes how to deploy Explorer locally by RPM and tar package.
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md:3]: by a RPM or TAR package/ by RPM and TAR packages

File: docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+
+## Nebula Graph version
+
+!!! Note
+
+    Explorer is released separately, not synchronized with Nebula Graph core. And the naming of Explorer is different from that of Nebula Graph. The version correspondence between Nebula Graph and Explorer is as follows.
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md:9]: with the Nebula Graph's kernel codebase?
or say " with Nebula Graph"

Comment (randomJoe211) [inline docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md:9]: naming of Explorer is different from that of Nebula Graph?
That is the fact... 
You mean version naming?

File: docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md [hunk 3]
```diff
@@ -75,63 +116,64 @@ Before you deploy Explorer, you must do a check of these:
...
+   For example:
    ```bash
-   $ cd nebula-graph-explorer
-   $ npm run start
+   $ cp -r nebula.license /usr/local/nebula-explorer
    ```
 
+  !!! enterpriseonly
+
+        License is only available in the Enterprise Edition. For more information, send an email to inquiry@vesoft.com.
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md:154]: send an email --->send email

File: docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md [hunk 4]
```diff
@@ -33,40 +44,70 @@ Before you deploy Explorer, you must do a check of these:
...
+
+   For example:
+   ```bash
+   $ cp -r nebula.license /usr/local/nebula-explorer
+   ```
+
+  !!! enterpriseonly
 
-### Exception handling
+        License is only available in the Enterprise Edition. For more information, send an email to inquiry@vesoft.com.
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md:71]: send email

**Review by izhuxiaoqing — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**